assembly/__tests__/BoolArray.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { BoolArray } from "../BoolArray";
+
+describe("Bool Array", () => {
```

Comment:
Can we add test case in which we check that if we provide invalid bytes (not true/false), the `BoolArray.fromU8a` throws?